File: js/Funções construtoras/Exemplo 1/Pessoa.js
function Pessoa() {
    let nome = ""; // Propriedade "privada"
    let idade = 0; // Propriedade "privada"
  
    // Métodos para acessar as propriedades
    this.setNome = function(novoNome) {
      nome = novoNome;
    };
  
    this.getNome = function() {
      return nome;
    };
  
    this.setIdade = function(novaIdade) {
      idade = novaIdade;
    };
  
    this.getIdade = function() {
      return idade;
    };
  }
  
  // Criando instâncias da função construtora
  const pessoa1 = new Pessoa();
  pessoa1.setNome("Ana"); // Define o nome
  console.log(pessoa1.getNome()); // "Ana"

File: py/classes e objetos/Exemplo 1/Pessoa.py
class Pessoa:
    def __init__(self,nome,idade):
        self.nome = nome
        self.idade=idade

    def DizerOla(self):
        print(f"Olá, meu nome é {self.nome} e tenho {self.idade} anos.")
    

pessoa1 = Pessoa("Claudio",45)
pessoa1.DizerOla()
        